lib/vtls/schannel.c

```diff
@@ -1743,8 +1749,6 @@ schannel_connect_common(struct Curl_cfilter *cf,
      * Available on Windows 7 or later.
      */
     {
-      struct schannel_ssl_backend_data *backend =
-        (struct schannel_ssl_backend_data *)connssl->backend;
       DEBUGASSERT(backend);
       cf->conn->sslContext = &backend->ctxt->ctxt_handle;
     }
```

Comment:
The `{}` block is no longer needed after this.

Reply:
The `DEBUGASSERT` could also be moved higher?

Reply:
Yes!

Reply:
Updated!